components/builder-web/index.html

```diff
@@ -1,11 +1,23 @@
 <!doctype html>
-<html>
+<html prefix="og: http://ogp.me/ns#">
   <head>
     <meta charset="utf-8">
     <meta http-equiv="x-ua-compatible" content="ie=edge">
     <title>Habitat - Automation that travels with the app.</title>
     <meta name="description" content="Habitat allows automation to travel with your app, so you can be confident that what you build, deploy and manage will behave consistently in any runtime.">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <meta property="og:title" content="Habitat - Automation that travels with the app.">
+    <meta property="og:type" content="website">
+    <meta property="og:image" content="/assets/images/habitat-social.jpg">
+    <meta property="og:url" content="https://www.habitat.sh/">
```

Comment:
@smith I don't know if it's important but all of the stats for shares under the app will be consolidated to this one URL. Don't know if there are Angularine ways to address that.